### ISSUE TITLE
FLVPlayback Component Broken?

### ISSUE THREAD

**Issue by EvilGrandpa420** [ruffle-rs/ruffle#17332]
### Describe the bug

When using FLVPlayback from the Components tab, The SWF will be blank in-browser and just show the scrub bar in ruffle desktop
Online:
![Screenshot 2024-08-01 175635](https://github.com/user-attachments/assets/afac696f-94a0-4cf8-92d9-4e17e892b228)
In Ruffle:
![Screenshot 2024-08-01 175516](https://github.com/user-attachments/assets/783d30ac-a1b0-47e2-9b44-cbe4b3340998)
In Adobe Flash Player 9:
![Screenshot 2024-08-01 175607](https://github.com/user-attachments/assets/9df886c0-11ff-45b8-9575-376cca9e1cfc)

### Expected behavior

I expected the video to play, instead of being blank

### Content Location

https://evilgrandpa.neocities.org/Videos/grannouncement

### Affected platform

Desktop app

### Operating system

Windows 11

### Browser

Opera GX 109.0.5097.142

**Comment by EvilGrandpa420** [comment]
Sorry, certified ActionScript dumbass here. I've now added the SkinOverPlaySeekFullscreen.swf and it still doesn't work? How do I show ruffle it's there?

**Comment by torokati44** [comment]
It does find and show the player skin for me now, just not quite in the right size/place:
![image](https://github.com/user-attachments/assets/76eb803d-f91b-4a55-8999-45c9338ab26c)

**Comment by EvilGrandpa420** [comment]
huh. that's quite strange! it still doesn't show up for me.

**Comment by torokati44** [comment]
Is music playing though?

**Comment by EvilGrandpa420** [comment]
Yes, the music is playing. The music is on a different layer to the video as the video is silent and feels uncomfortable without the music.

**Comment by torokati44** [comment]
> huh. that's quite strange! it still doesn't show up for me.

It takes a little while for me too to load. A second or two...
